compositor_pipeline/src/pipeline/input/rtp/depayloader.rs

```diff
@@ -101,24 +98,23 @@ impl VideoDepayloader {
                     return Ok(None);
                 }
 
-                if *last_timestamp == 0 {
-                    *last_timestamp = packet.header.timestamp;
-                }
-
-                if packet.header.timestamp == *last_timestamp {
+                if !packet.header.marker {
+                    // not at access unit border, buffer the packet
                     buffer.push(h264_chunk);
```

Comment:
```suggestion
                buffer.push(h264_chunk);
                if !packet.header.marker {
                    // Marker is set on the last packet of an access unit.
```
I assume my comment suggestion is correct based on your implementation (I did not check that)